en/docs/learn/multi-attribute-login.md

```diff
@@ -7,10 +7,13 @@ how to configure this.
 
 ## Configuring WSO2 IS for Multi Attribute Login
 
-1. Start the Identity Server and log in to the management console using your tenant credentials.
+1. Start the Identity Server and log in to the [Management Console](https://localhost:9443/carbon) using your tenant credentials.
```

Comment:
Shall we use this instead of https://localhost:9443/carbon? 

`https://<IS_HOST>:<PORT>/carbon`

Reply:
Fixed: abde67e